### ISSUE TITLE
Setup with no pieces

### ISSUE THREAD

**Issue by RoniJacobson** [itamarLaufer/SuperTactico#38]
I think it may be a good idea to have an option to get the pieces without them being put on the board, and having the option to drag them on. This would make setting up the board in the beginning of the game more convenient.

**Comment by itamarLaufer** [comment]
Obviously, It's more intuitive. I can support this in the server. Do you have anything in mind about how it would look like? I have something classic but it may take a serious amount of time. We should start keeping in mind that the time we have to work on it together (when we are on the same place) is limited. You may take a look on #21, there are a few issues which are not quite easy and maybe it's a good idea to finish with them first..

**Comment by itamarLaufer** [comment]
The logic was supported in a local branch but won't go to production until it will be implemented in the GUI.

**Comment by itamarLaufer** [comment]
At first the client gets the pieces as it has always been except all the locations will be `[-2, -2].`
Asking for possible orders of piece outside the board (represented by `[-2, -2]` location) will return all the possible locate orders on the board exactly like in the previous setup.
For a piece that is on the board, a locate order to `[-2, -2]` (outside the board) will be added, indexed with `0`.
After each locate order an attribute will be sent that is called `allSet `which will be set to True if all pieces are on the board and the client can search for an opponent.
